### PR TITLE
Add `osu! Austria @ Conventions 2024` news post

### PR DESCRIPTION
**Pull request by yasuhosu**
this is a news post about the presence of the osu! Austria community at upcoming conventions in 2024. also the announcement of LAN tournaments at said conventions.

[web preview](https://osu1.roanh.dev/home/news/2024-07-23-osu-austria-conventions-2024)

### CODE REVIEW

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/Banner.png)
+
+The Austrian osu! community is excited to announce their presence at two upcoming conventions. Next to osu! booths, there will also be LAN tournaments, in which you can show off your skills and win some nice goodies. So what are you waiting for? Get all the information you need down below!
+
+## AniNite 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)
+
+Join us at the biggest Anime and Manga convention in Austria, [Aninite](https://www.aninite.at/en/) has over 25.000 yearly visitors, meet our community on the **9th to 11th of August 2024** at our osu! booth. In coorperation with [Friendly Fire](https://friendlyfireesports.com/), we organized 12 PCs for you to play on and show what you are made of.
+During this time, we will host a tournament, **Battle Royale** style. Register live on the spot and take home the trophy!
```
Comment (Hiviexd) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:18]:
```suggestion
During this time, a **battle royale-styled** osu! tournament will be hosted. Make sure to register during the event and take home the trophy!
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## YuniCon 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/).
```
Comment (Hiviexd) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:24]:
```suggestion
Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers provided by [Streamhouse](https://streamhouse.at/)!
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+On all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers.
```
Comment (Hiviexd) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:25]:
```suggestion
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Computers are being provided by [Streamhouse](https://streamhouse.at/).
```
Comment (Hiviexd) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:26]:
```suggestion
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+On top of that, there will also be the biggest osu! 1v1 LAN tournament Austria has ever seen! The top 2 players will fight it out for real life prizes and a profile badge on the main stage of the event.
```
Comment (Hiviexd) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:28]:
```suggestion
On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament Austria has ever seen! The top players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event.
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+| Quarterfinals | 2024-11-02 (14:30-17:00) |
+| Semifinals | 2024-11-02 (17:30-20:00) |
+| Finals | 2024-11-03 (10:00-13:00) |
+| Grandfinals | 2024-11-03 (13:15) |
+
+---
+
+For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.gg/dNFuHNmZF4) or follow [@osuaustria on Twitter](https://twitter.com/osuaustria)
+
+We can’t wait to see you there!
```
Comment (Hiviexd) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:49]:
```suggestion
We can't wait to see you there!
```

**Review by Hiviexd — CHANGES_REQUESTED** [review]

**Review by Hiviexd — CHANGES_REQUESTED** [review]
the transparency and centering of [aninite.png](https://github.com/ppy/osu-wiki/pull/11802/files#diff-1326a4151ed61c3daad293c191cc1b1faa04b295775516a657acb63d0b629ba1) and [yunicon.png](https://github.com/ppy/osu-wiki/pull/11802/files#diff-bb6a4e4f4dc82c927a97686bf5fef70eb1a1284720d87af8a94e6de11e60cf17) looks pretty messed up, consider fixing up via making the grey background transparent, and actually centering the banners on the Y axis.

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+On top of that, there will also be the biggest osu! 1v1 LAN tournament Austria has ever seen! The top 2 players will fight it out for real life prizes and a profile badge on the main stage of the event.
+Top 16 players won't go home empty-handed as they will receive some nice goodies as well!
+
+Join our [osu! Austrian Tournaments discord server](https://discord.gg/vRbC3bw) to not miss out on any updates regarding this tournament.
+
+### Tournament schedule
+
+| Event | Timestamp (UTC+1) |
+| --: | :-- |
+| Registrations | 2024-10-31 (13:00-23:00) |
```
[the remark this answers is not in the document]
Comment (yasuhosu) [reply news/2024/2024-07-23-osu-austria-conventions-2024.md]: did

**Comment by 0x84f** [comment]
for aninite it doesnt say when the tournament is taking place, is it all happening in 1 day? also for yunicon it doesnt say what type of tournament it will be

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+During this time, a **battle royale-styled** osu! tournament will be hosted. Make sure to register during the event and take home the trophy!
+
+## YuniCon 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers provided by [Streamhouse](https://streamhouse.at/)!
+
+On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament Austria has ever seen! The top players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event.
+Top 16 players won't go home empty-handed as they will receive some nice goodies as well!
```
Comment (0x84f) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:28]: this doesnt have to be a new sentence, it could be tacked onto the existing one

**Comment by yasuhosu** [comment]
> also for yunicon it doesnt say what type of tournament it will be

@0x84f it says 1v1 LAN tournament, should be sufficient?

**Comment by yasuhosu** [comment]
> the transparency and centering of [aninite.png](https://github.com/ppy/osu-wiki/pull/11802/files#diff-1326a4151ed61c3daad293c191cc1b1faa04b295775516a657acb63d0b629ba1) and [yunicon.png](https://github.com/ppy/osu-wiki/pull/11802/files#diff-bb6a4e4f4dc82c927a97686bf5fef70eb1a1284720d87af8a94e6de11e60cf17) looks pretty messed up, consider fixing up via making the grey background transparent, and actually centering the banners on the Y axis.

forwarded this to the designer of the banners, expect a fix soon.

**Comment by 0x84f** [comment]
> > also for yunicon it doesnt say what type of tournament it will be
> 
> @0x84f it says 1v1 LAN tournament, should be sufficient?

err i guess, except for the schedule though we dont know if its a bracket based tournament? should be explicitly stated

File: news/2024/2024-07-23-osu-austria-conventions-2024.md (hunk 8 above)
Comment (yasuhosu) [reply news/2024/2024-07-23-osu-austria-conventions-2024.md]: did something here

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/Banner.png)
+
+The Austrian osu! community is excited to announce their presence at two upcoming conventions. Next to osu! booths, there will also be LAN tournaments, in which you can show off your skills and win some nice goodies. So what are you waiting for? Get all the information you need down below!
+
+## AniNite 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)
+
+Join us at the biggest Anime and Manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. Meet our community from **August 9th to 11th, 2024** at our osu! booth. In cooperation with [Friendly Fire](https://friendlyfireesports.com/), we have organized 12 PCs for you to play on and show everyone what you're made of.
```
Comment (iSlodinxOsu) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:17]:
```suggestion
Join us at the biggest Anime and Manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. Meet our community from **August 9th to 11th, 2024** at our osu! booth. In cooperation with [Friendly Fire](https://friendlyfireesports.com/), we have organised 12 PCs for you to play on and show everyone what you're made of.
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+During this time, a **battle royale-styled** osu! tournament will be hosted. Make sure to register during the event and take home the trophy!
+
+## YuniCon 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organized computers provided by [Streamhouse](https://streamhouse.at/)!
```
Comment (iSlodinxOsu) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:25]:
```suggestion
Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers provided by [Streamhouse](https://streamhouse.at/)!
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

**Comment by yasuhosu** [comment]
> the transparency and centering of [aninite.png](https://github.com/ppy/osu-wiki/pull/11802/files#diff-1326a4151ed61c3daad293c191cc1b1faa04b295775516a657acb63d0b629ba1) and [yunicon.png](https://github.com/ppy/osu-wiki/pull/11802/files#diff-bb6a4e4f4dc82c927a97686bf5fef70eb1a1284720d87af8a94e6de11e60cf17) looks pretty messed up, consider fixing up via making the grey background transparent, and actually centering the banners on the Y axis.

has been fixed in latest commit.

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+The Austrian osu! community is excited to announce their presence at two upcoming conventions. Next to osu! booths, there will also be LAN tournaments, in which you can show off your skills and win some nice goodies. So what are you waiting for? Get all the information you need down below!
+
+## AniNite 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)
+
+Join us at the biggest Anime and Manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. Meet our community from **August 9th to 11th, 2024** at our osu! booth. In cooperation with [Friendly Fire](https://friendlyfireesports.com/), we have organised 12 PCs for you to play on and show everyone what you're made of.
+
+During this time, a **battle royale-styled** osu! tournament will be hosted. Make sure to register during the event and take home the trophy!
```
Comment (Walavouchey) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:19]: en dash when it connects terms with spaces between them

```suggestion
During this time, we'll host a **battle royale–styled** osu! tournament. Make sure to register during the event and take home the trophy!
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+## YuniCon 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers provided by [Streamhouse](https://streamhouse.at/)!
```
Comment (Walavouchey) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:25]:
```suggestion
Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+
+On top of that, you'll also be able to witness the biggest osu! single elimination bracket based 1v1 LAN tournament Austria has ever seen! 
+You'll be able to sign-up for the tournament online. Please join [our Discord server](https://discord.gg/dNFuHNmZF4), to get notified when sign-ups open.
+Qualifiers will be played out online as well, scheduling your lobby will also be handled over at our **Discord server**. After you qualified for the round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!
```
Comment (Walavouchey) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:29]:
```suggestion
On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!

You'll be able to sign-up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.

Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*. After you qualified for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+
+On top of that, you'll also be able to witness the biggest osu! single elimination bracket based 1v1 LAN tournament Austria has ever seen! 
+You'll be able to sign-up for the tournament online. Please join [our Discord server](https://discord.gg/dNFuHNmZF4), to get notified when sign-ups open.
+Qualifiers will be played out online as well, scheduling your lobby will also be handled over at our **Discord server**. After you qualified for the round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents! 
+
+The top 2 players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event and top 16 players won't go home empty-handed as they will receive some nice goodies as well!
```
Comment (Walavouchey) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:31]:
```suggestion
The top 2 players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event, and the top 16 players won't go home empty-handed either as they will receive some nice goodies as well!
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 15]
```diff
@@ -0,0 +1,51 @@
...
+
+| Event | Timestamp (UTC+1) |
+| --: | :-- |
+| Registrations | 2024-10-31 (13:00–23:00) |
+| Qualifiers | 2024-11-01 (10:00–19:00) |
+| Round of 16 | 2024-11-02 (10:00–14:00) |
+| Quarterfinals | 2024-11-02 (14:30–17:00) |
+| Semifinals | 2024-11-02 (17:30–20:00) |
+| Finals | 2024-11-03 (10:00–13:00) |
+| Grandfinals | 2024-11-03 (13:15) |
```
Comment (Walavouchey) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:43]: for news posts we usually write dates out for readability

```suggestion
| Registrations | 31 October 2024 (13:00–23:00) |
| Qualifiers | 1 November 2024 (10:00–19:00) |
| Round of 16 | 2 November 2024 (10:00–14:00) |
| Quarterfinals | 2 November 2024 (14:30–17:00) |
| Semifinals | 2 November 2024 (17:30–20:00) |
| Finals | 3 November 2024 (10:00–13:00) |
| Grand Finals | 3 November 2024 (13:15) |
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 16]
```diff
@@ -0,0 +1,51 @@
...
+
+---
+
+For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.gg/dNFuHNmZF4) and follow [@osuaustria on Twitter](https://twitter.com/osuaustria)
```
Comment (Walavouchey) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:47]:
```suggestion
For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.com/invite/dNFuHNmZF4) and follow [@osuaustria on Twitter](https://twitter.com/osuaustria)
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by peppy — DISMISSED** [review]

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 17]
```diff
@@ -0,0 +1,53 @@
...
+
+On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!
+
+You'll be able to sign-up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.
+
+Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*. After you qualified for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!
+
+The top 2 players of the tournament will ferociously duke it out for some handsome physical prizes and a bespoke profile badge on the main stage of the event, and the top 16 players won't go home empty-handed either as they will receive some nice goodies as well!
```
Comment (0x84f) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:33]: I assume the physical prizes will only go to the winner? rewrote this the way i liked it more, but its subjective ofc
```suggestion
The top 2 players of the tournament will ferociously duke it out on the main stage of the event, with the winner receiving handsome physical prizes and a bespoke profile badge. The top 16 players won't go home empty-handed either, as they will receive some nice goodies as well!
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 18]
```diff
@@ -0,0 +1,53 @@
...
+| Qualifiers | 1 November 2024 (10:00–19:00) |
+| Round of 16 | 2 November 2024 (10:00–14:00) |
+| Quarterfinals | 2 November 2024 (14:30–17:00) |
+| Semifinals | 2 November 2024 (17:30–20:00) |
+| Finals | 3 November 2024 (10:00–13:00) |
+| Grand Finals | 3 November 2024 (13:15) |
+
+---
+
+For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.com/invite/dNFuHNmZF4) and follow [@osuaustria on Twitter](https://twitter.com/osuaustria)
```
Comment (0x84f) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:49]:
```suggestion
For all the latest news on the happenings throughout the event, join the [osu! Austria meetups & events Discord server](https://discord.com/invite/dNFuHNmZF4) and follow [@osuaustria on Twitter](https://twitter.com/osuaustria).
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 19]
```diff
@@ -0,0 +1,53 @@
...
+
+## YuniCon 2024
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at [Yunicon 2024](https://www.yunicon.at/en/). During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!
+
+On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!
+
+You'll be able to sign-up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.
```
Comment (0x84f) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:29]:
```suggestion
You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 20]
```diff
@@ -14,18 +14,20 @@ Next to osu! booths, there will also be LAN tournaments, in which you can show o
 
 ![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/aninite.png)
 
-Join us at the biggest anime and manga convention in Austria, [AniNite](https://www.aninite.at/en/), which has over 25,000 yearly visitors. You can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.
+Join us at the biggest anime and manga convention in Austria with over 25,000 visitors each year: **[AniNite](https://www.aninite.at/en/)**! Located near the beautiful danube at Austria Center in Vienna. You can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.
```
Comment (0x84f) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:17]:
```suggestion
Join us at the biggest anime and manga convention in Austria with over 25,000 visitors each year: **[AniNite](https://www.aninite.at/en/)**! Located near the beautiful Danube River at Austria Center in Vienna. You can find us from **August 9th to 11th** at our osu! booth, where there will be 12 PCs for you to play on and show everyone what you're made of.
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 21]
```diff
@@ -0,0 +1,55 @@
...
+
+Next up, we're setting up an osu! booth at **[Yunicon 2024](https://www.yunicon.at/en/)**, situated at Mutliversum Schwechat. During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!
+
+On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicupbannerfinals.png)
+
+You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.
+
+Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*. After you qualified for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!
```
Comment (0x84f) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:33]: `Qualifiers will be played out online as well and scheduling your lobby will be handled over at our *Discord server*`
not sure if this needs to be italicised? i'd either remove the italicising or that sentence altogether and merge it with the one above where the discord server is actually linked

Comment (yasuhosu) [reply news/2024/2024-07-23-osu-austria-conventions-2024.md:33]: ermm i dont even remember why i italicized that. the reason why i'd want to keep that, is to make sure people know how it works with the scheduling. rather one question less here. but ill remove the italicization.

File: news/2024/2024-07-23-osu-austria-conventions-2024.md [hunk 22]
```diff
@@ -0,0 +1,55 @@
...
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicon.png)
+
+Next up, we're setting up an osu! booth at **[Yunicon 2024](https://www.yunicon.at/en/)**, situated at Mutliversum Schwechat. During all 4 days of the convention, you'll be able to meet our community, ask questions and play on 10 pre-organised computers!
+
+On top of that, you'll also be able to witness the biggest osu! 1v1 LAN tournament with a single-elimination bracket that Austria has ever seen!
+
+![](/wiki/shared/news/2024-07-23-osu-austria-conventions-2024/yunicupbannerfinals.png)
+
+You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) to get notified when sign-ups open.
```
Comment (0x84f) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:31]: In relation to [this](https://github.com/ppy/osu-wiki/pull/11802/files#r1688447290):
```suggestion
You'll be able to sign up for the tournament online. Please join our [Discord server](https://discord.com/invite/dNFuHNmZF4) where you can get notified when sign-ups open, and where organisation of the Qualifiers will happen.
```

File: news/2024/2024-07-23-osu-austria-conventions-2024.md (hunk 21 above)
Comment (0x84f) [inline news/2024/2024-07-23-osu-austria-conventions-2024.md:33]:
```suggestion
After you qualify for the Round of 16, **you're required to show up**. A no-show will automatically **eliminate** you from the tournament and open your spot for the next possible seed that is present at the event. Bring the peripherals you're comfortable with and prepare to bash your opponents!
```

Comment (0x84f) [reply news/2024/2024-07-23-osu-austria-conventions-2024.md:33]: check out what i just suggested

Comment (yasuhosu) [reply news/2024/2024-07-23-osu-austria-conventions-2024.md]: applied all

**Review by 0x84f — APPROVED** [review]

**Review by Walavouchey — DISMISSED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**